portfolio/src/main/webapp/index.html

```diff
@@ -2,14 +2,15 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Connor's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+    <div class="card">
+      <h1 fontSize = "6vw">Connor's Portfolio</h1>
```

Comment:
One more nit: According to https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns, let's keep styling in css files and have the HTML only be for structure purposes.
Same for the line below for your avatar.

Reply:
is there a problem with line 10 or just line 11?

Reply:
Huh, I don't know why github showed both line 10 and 11. The comment is actually about lines 11 and 12 lol

Reply:
I understand the issue with this line: " <h1 fontSize = "6vw">Connor's Portfolio</h1>". Is there an issue with this line: "<div class="card">"?